public/app/features/alerting/unified/components/export/ExportNewGrafanaRule.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { AlertingPageWrapper } from '../AlertingPageWrapper';
+import { ModifyExportRuleForm } from '../rule-editor/alert-rule-form/ModifyExportRuleForm';
+
+export default function ExportNewGrafanaRule() {
+  return (
+    <ExportNewGrafanaRuleWrapper>
+      <ModifyExportRuleForm />
+    </ExportNewGrafanaRuleWrapper>
+  );
+}
+
+interface ExportNewGrafanaRuleWrapperProps {
+  children: React.ReactNode;
+}
+
+function ExportNewGrafanaRuleWrapper({ children }: ExportNewGrafanaRuleWrapperProps) {
+  return (
+    <AlertingPageWrapper
+      navId="alert-list"
+      pageNav={{
+        text: 'Export new Grafana rule',
+        subTitle: 'Export a new rule definition in the format of your choice. Any changes you make will not be saved.',
```

Comment:
Should we adjust the message and be explicit that only Terraform is supported?
Like `Export a new rule definition in Terraform(HCL) format. Any changes....`?